esp-hal/src/dma/mod.rs

```diff
@@ -1,20 +1,17 @@
-//! # Direct Memory Access
+//! # Direct Memory Access (DMA)
 //!
 //! ## Overview
-//!
-//! The `DMA` driver provides an interface to efficiently transfer data between
+//! The DMA driver provides an interface to efficiently transfer data between
 //! different memory regions and peripherals within the ESP microcontroller
-//! without involving the CPU. The `Direct Memory Access` (DMA) controller is a
-//! hardware block responsible for managing these data transfers.
+//! without involving the CPU. The DMA controller is a hardware block
+//! responsible for managing these data transfers.
```

Comment:
```suggestion
//! without involving the CPU. The DMA controller is responsible for
//! managing these data transfers.
```

This sounds a bit awkward to me, and can be simplified IMO.